<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spectrumParametersSelection</class>
 <widget class="QDialog" name="spectrumParametersSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>spectrumParametersSelection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="4">
      <widget class="QGroupBox" name="subbandGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Subbands</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>4S</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_8">
            <property name="text">
             <string>1L</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox_7">
            <property name="text">
             <string>4M</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>1S</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>2S</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_6">
            <property name="text">
             <string>2M</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>3S</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_16">
            <property name="text">
             <string>3M</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBox_14">
            <property name="text">
             <string>4L</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_10">
            <property name="text">
             <string>2L</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_12">
            <property name="text">
             <string>3L</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>1M</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="spectrumComboBox">
       <item>
        <property name="text">
         <string>Aperture spectrum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Background spectrum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Background subtracted spectrum</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Spectrum type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="fUnitsComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>erg/s/cm2/um</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mJy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uJy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W/m2/Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>erg/s/cm2/Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>W/m2/Angstroms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>erg/s/cm2/Angstroms</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="wUnitsComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>um</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>angstroms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nm</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Wavelength units</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Flux units</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QPushButton" name="acceptButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Subband</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="allRadioButton">
       <property name="text">
        <string>All subbands</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="contiguousRadioButton">
       <property name="text">
        <string>Contiguous subbands</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>spectrumParametersSelection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>spectrumParametersSelection</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
